Style.css

```diff
@@ -884,6 +865,12 @@ body {
   border: none;
   width: 131px;
   height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
```

Comment:
- [ ] **[OPTIONAL]** I have a suggestion that might help fix the issue with horizontal scrolling. You could consider removing the fixed widths you added, such as the 375px width in the mobile-container. Instead, I would suggest changing it to 100%. The same change could be applied to the other elements as well. 🤓

   If you'd like to identify which elements are extending beyond your borders, you can use the following class to create a border around all your elements. This can make debugging your project easier:

   ```
   * {
     border: 1px solid red;
   }
   ```